datastore/datastore.go

```diff
@@ -28,6 +30,9 @@ type DataStore struct {
 
 	cacheSyncs []cache.InformerSynced
 
+	proxyConnectionLock *sync.RWMutex
+	proxyConnection     int
```

Comment:
Can we launch a separate counter struct in `app/daemon.go` instead?
I prefer not to ask `datastore` to record this.

Besides, we can use functions in package `atomic` to operate the int.

Reply:
I cannot put it in the pkg app because of the import loop. After talking to @shuo-wu , we will put it in pkg util.